core/src/main/scala/kafka/server/ControllerMutationQuotaManager.scala

```diff
@@ -131,6 +133,16 @@ class PermissiveControllerMutationQuota(private val time: Time,
 
 object ControllerMutationQuotaManager {
   val QuotaControllerMutationDefault = Int.MaxValue.toDouble
+
+  def throttleTime(e: QuotaViolationException, timeMs: Long): Long = {
+    e.metric().measurable() match {
+      case _: TokenBucket => Math.round(-e.value() * e.bound())
```

Comment:
Hmm, assuming bound is the per sec rate, it seems that the throttleTimeMs should be `-e.value() / e.bound * 1000`?

Reply:
Yes, that's right.